measures/ResStockArguments/measure.rb

```diff
@@ -356,58 +356,6 @@ def run(model, runner, user_arguments)
       end
     end
 
-    # Misc LULs
-    constant = 1.0 / 2
-    nbr_coef = 1.0 / 4 / 3
-    ffa_coef = 1.0 / 4 / 1920
-
-    ['misc_plug_loads_well_pump_annual_kwh',
-     'misc_fuel_loads_grill_annual_therm',
-     'misc_fuel_loads_lighting_annual_therm',
-     'misc_fuel_loads_fireplace_annual_therm',
-     'pool_pump_annual_kwh',
-     'pool_heater_annual_kwh',
-     'pool_heater_annual_therm',
-     'hot_tub_pump_annual_kwh',
-     'hot_tub_heater_annual_kwh',
-     'hot_tub_heater_annual_therm'].each do |annual_energy|
-      next if args[annual_energy] != Constants.Auto
-
-      if annual_energy == 'misc_plug_loads_well_pump_annual_kwh'
-        args[annual_energy] = 50.8 / 0.127
-      elsif annual_energy == 'misc_fuel_loads_grill_annual_therm'
-        args[annual_energy] = 0.87 / 0.029
-      elsif annual_energy == 'misc_fuel_loads_lighting_annual_therm'
-        args[annual_energy] = 0.22 / 0.012
-      elsif annual_energy == 'misc_fuel_loads_fireplace_annual_therm'
-        args[annual_energy] = 1.95 / 0.032
-      elsif annual_energy == 'pool_pump_annual_kwh'
-        args[annual_energy] = 158.6 / 0.070
-      elsif annual_energy == 'pool_heater_annual_kwh'
-        args[annual_energy] = 8.3 / 0.004
-      elsif annual_energy == 'pool_heater_annual_therm'
-        args[annual_energy] = 3.0 / 0.014
-      elsif annual_energy == 'hot_tub_pump_annual_kwh'
-        args[annual_energy] = 59.5 / 0.059
-      elsif annual_energy == 'hot_tub_heater_annual_kwh'
-        args[annual_energy] = 49.0 / 0.048
-      elsif annual_energy == 'hot_tub_heater_annual_therm'
-        args[annual_energy] = 0.87 / 0.011
-      end
-
-      if [HPXML::ResidentialTypeSFD].include?(args['geometry_unit_type'])
-        args[annual_energy] *= (constant + nbr_coef * (-1.47 + 1.69 * args['geometry_unit_num_occupants']) + ffa_coef * args['geometry_unit_cfa'])
-      elsif [HPXML::ResidentialTypeSFA, HPXML::ResidentialTypeApartment].include?(args['geometry_unit_type'])
-        args[annual_energy] *= (constant + nbr_coef * (-0.68 + 1.09 * args['geometry_unit_num_occupants']) + ffa_coef * args['geometry_unit_cfa'])
-      end
-
-      if annual_energy == 'pool_heater_annual_kwh' && args['pool_heater_type'] == HPXML::HeaterTypeHeatPump
-        args[annual_energy] /= 5.0
-      elsif annual_energy == 'hot_tub_heater_annual_kwh' && args['hot_tub_heater_type'] == HPXML::HeaterTypeHeatPump
-        args[annual_energy] /= 5.0
-      end
```

Comment:
Are these accounted for?

Reply:
Yes. In fact, I think they originally came from the OS-HPXML default assumptions (see [here](https://openstudio-hpxml.readthedocs.io/en/latest/workflow_inputs.html#id702))?